src/commands/dsyms/utils.ts

```diff
@@ -0,0 +1,127 @@
+import {exec} from 'child_process'
+import {mkdirSync} from 'fs'
+import glob from 'glob'
+import path from 'path'
+import {promisify} from 'util'
+import {buildPath} from '../../helpers/utils'
+import {Payload} from './interfaces'
+import {checkNonEmptyFile} from './validation'
+
+const UUID_REGEX = '[0-9A-F]{8}-([0-9A-F]{4}-){3}[0-9A-F]{12}'
+
+const getMatchingDSYMFiles = async (absoluteFolderPath: string): Promise<Payload[]> => {
+  const dSYMFiles = glob.sync(buildPath(absoluteFolderPath, '**/*.dSYM'))
```

Comment:
NIT: Why using `.sync` here?

Reply:
i took this from `getMatchingSourcemapFiles` in `sourcemaps/upload.ts`
i can make it `async`, should i?

Reply:
Since the function is `async` already, I think it's safe to use async/await code in it.
It just felt out of place here, I won't block for it.